src/jdk.jdwp.agent/share/native/libjdwp/util.h

```diff
@@ -65,7 +65,10 @@ typedef struct RefNode {
     jobject      ref;           /* could be strong or weak */
     struct RefNode *next;       /* next RefNode* in bucket chain */
     jint         count;         /* count of references */
-    unsigned     strongCount;   /* count of strong reference */
+    jboolean     isStrong;      /* true if this is a strong reference. Either or both of isPinAll
+                                   and isCommonPin will also be true */
```

Comment:
Doesn't that mean we don't need the `isStrong` field but rather:
```
jboolean isStrong() { return isPinAll || isCommonPin; }
```
?

Reply:
I considered doing that. I can still make that change if you think it would be better.

Reply:
I think it would be simpler/clearer.